source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs

```diff
@@ -292,6 +295,24 @@ public IActionResult GetAcquisitionFileSubFiles(long id)
             return new JsonResult(_mapper.Map<List<AcquisitionFileModel>>(subFiles));
         }
 
+        /// <summary>
+        /// Gets a collection of documents for the specified type and owner id.
+        /// </summary>
+        /// <param name="id">Used to identify document type.</param>
+        /// <param name="time">Used to identify document's parent entity.</param>
```

Comment:
param names are correct but the descriptions maybe not?

Reply:
will remove the comments all-together; not sure they add much value